include/bluetooth/mesh/scene_srv.h

```diff
@@ -49,6 +49,17 @@ extern "C" {
 
 struct bt_mesh_scene_srv;
 
+/** @def BT_MESH_SCENE_SRV_INIT
+ *
+ * @brief Initialization parameters for @ref bt_mesh_scene_srv.
+ *
+ * @param[in] _dtt_srv      Default Transition Time Server instance.
+ */
+#define BT_MESH_SCENE_SRV_INIT(_dtt_srv)                                       \
```

Comment:
That makes sense, but then should Gen-ponoff-srv add DTT srv or should we also expect User to do that and then just use `bt_mesh_dtt_srv_get`?

Reply:
While adding MMDLs to the `tester` app it happened to me a few times that there were multiple instances of the same model on one element. Might make sense to verify that for the user on startup?

Reply:
Maybe keep Generic Power OnOff implementation as is for now and use `bt_mesh_dtt_srv_get` in Scene server? If `bt_mesh_dtt_srv_get` returns NULL in Scene server, we can add error log. And `bt_mesh_init` will fail in this case. But if Generic Power OnOff is on the same element, it would work without any changes from a user. This will also require a doc update for Scene server model.

Yeah, I think we have to add verification for the same models/messages on the same element.